ios/Plugin/TwilioVideoViewController.swift

```diff
@@ -2,6 +2,17 @@ import UIKit
 import Foundation
 import TwilioVideo
 
+enum TwilioEvent: String {
```

Comment:
just curious, I suppose that Android uses the very same strings (I HOPE), so would it be possible to have some kind of common config for all supported platforms? does capacitor offer that?

Reply:
yes, they are the same strings. (I HOPE SO TOO). Capacitor doesn't offer that. However we may achieve this using a common json/xml file.

Reply:
It's not documented but Capacitor offers a help class to read values from capacitor.config.json which can be used as a key-value config store.

Reply:
we can do it in a follow up task to factor out common stuff

Reply:
great, yes!
https://media4care.atlassian.net/browse/NG-1241